src/cow-react/modules/wallet/api/pure/WalletModal/index.tsx

```diff
@@ -61,6 +73,7 @@ export function WalletModal(props: WalletModalProps) {
                   <ConnectWalletOptions tryActivation={tryActivation} />
                 </OptionGrid>
               )}
+              {!account && <ZengoBanner />}
```

Comment:
does it make sense to also require to not have `window.ethereum`?

I would think that if a user has `window.ethereum` in their browse, they don't need that banner

Reply:
Oh yea, good point. Will add that